extensions/parquet/table/src/main/java/io/deephaven/parquet/table/ParquetInstructions.java

```diff
@@ -178,6 +213,20 @@ public final String getColumnNameFromParquetColumnNameOrDefault(final String par
      */
     public abstract boolean generateMetadataFiles();
 
+    public abstract ParquetFileLayout getFileLayout();
+
+    public abstract TableDefinition getTableDefinition();
```

Comment:
(Comment from #5206:)
Should these be Optional? Maybe that's a general question for configs with null defaults.